inc/functions.php

```diff
@@ -5156,7 +5156,7 @@ function convert_through_utf8($str, $to=true)
 }
 
 /**
- * DEPRECATED! Please use other alternatives.
+ * @deprecated deprecated since version 1.8.4 Please use other alternatives.
```

Comment:
https://github.com/mybb/mybb/pull/2027/files#diff-7fe6e55397c77ab9a0f5d57bc4cbe5b9R5198

The function still has a parameter and returns something